### PR TITLE
Provides a release_template function - generates template from operator

### PR DESCRIPTION
**Pull request by phantomjinx**

### CODE REVIEW

**Comment by phantomjinx** [comment]
Not sure how much more work is requires so left it WIP for now.

**Comment by phantomjinx** [comment]
Cannot be merged until the 1.8 release of the operator extracted from docker contains the <install forge> command. Without that the script with produce an error.

File: release.sh [hunk 1]
```diff
@@ -286,6 +288,77 @@ extract_binaries() {
...
+
+    #
+    # docker uses the 'darwin' directory rather than mac
+    #
+    osdir=${CURRENT_OS}
+    if [ "$osdir" == "mac" ]; then
+      osdir="darwin"
+    fi
+
+    local tmp_dir=/tmp/fuse-template
```
Comment (lgarciaaco) [inline release.sh]: what about using `mktemp`?

File: release.sh [hunk 2]
```diff
@@ -286,6 +288,77 @@ extract_binaries() {
...
+    pushd $syndesis_dir > /dev/null
+    if [ -f "${SYNDESIS_BINARY}.gz" ]; then
+        if [ $osdir == "windows" ]; then
+          gunzip "${SYNDESIS_BINARY}.gz" && mv ${SYNDESIS_BINARY} ${SYNDESIS_BINARY}.exe
+          SYNDESIS_BINARY="${SYNDESIS_BINARY}.exe"
+        else
+          gunzip "${SYNDESIS_BINARY}.gz"
+        fi
+    fi
+    set -e
```
Comment (lgarciaaco) [inline release.sh]: Cant we just close the `set +e` at the end of this script?

File: release.sh (hunk 1 above)
Comment (phantomjinx) [reply release.sh]: done

File: release.sh [hunk 3]
```diff
@@ -286,6 +288,77 @@ extract_binaries() {
...
+
+    if [ ! -f $SYNDESIS_BINARY ]; then
```
[the remark this answers is not in the document]
Comment (phantomjinx) [reply release.sh]: done.

**Pull request merged**